platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/TlsFactory.java

```diff
@@ -51,21 +53,29 @@ public class TlsFactory implements SocketFactory {
     private final SSLSocketFactory sslSocketFactory;
 
     /**
-     * Construct this object to create and receive TLS connections. This is done using the trustStore
-     * whose reference was passed in as an argument. That trustStore must contain certs for all
-     * the members before calling this constructor. This method will then create the appropriate
-     * KeyManagerFactory, TrustManagerFactory, SSLContext, SSLServerSocketFactory, and SSLSocketFactory, so
-     * that it can later create the TLS sockets.
+     * Construct this object to create and receive TLS connections.
+     * @param agrCert the TLS certificate to use
+     * @param agrKey the private key corresponding to the public key in the certificate
+     * @param peers the list of peers to allow connections with
+     * @param socketConfig the configuration for the sockets
+     * @param cryptoConfig the configuration for the cryptography
      */
     public TlsFactory(
-            @NonNull final KeysAndCerts keysAndCerts,
+            @NonNull final Certificate agrCert,
+            @NonNull final PrivateKey agrKey,
+            @NonNull final List<PeerInfo> peers,
             @NonNull final SocketConfig socketConfig,
             @NonNull final CryptoConfig cryptoConfig)
             throws NoSuchAlgorithmException, UnrecoverableKeyException, KeyStoreException, KeyManagementException,
                     CertificateException, IOException {
-        Objects.requireNonNull(keysAndCerts);
-        Objects.requireNonNull(cryptoConfig);
+        Objects.requireNonNull(agrCert);
+        Objects.requireNonNull(agrKey);
+        Objects.requireNonNull(peers);
         this.socketConfig = Objects.requireNonNull(socketConfig);
+        Objects.requireNonNull(cryptoConfig);
+
+        // final KeyStore signingTrustStore = keysAndCerts.publicStores().sigTrustStore();
```

Comment:
commented line of code remains.  Explain why it is there, or delete it.

Reply:
left it there by mistake, removed